content/200-concepts/050-overview/100-what-is-prisma/index.mdx

```diff
@@ -20,7 +20,12 @@ Prisma is an [open source](https://github.com/prisma/prisma) next-generation ORM
 
 - **Prisma Client**: Auto-generated and type-safe query builder for Node.js & TypeScript
 - **Prisma Migrate**: Migration system
-- **Prisma Studio**: GUI to view and edit data in your database
+- **Prisma Studio**: GUI to view and edit data in your database.
+  <Admonition>
+
+  **Prisma Studio** is the only part of Prisma ORM that is not open source. It is fully integrated in our commercial offering [Prisma Data Platform](/data-platform/about) under the name **Data Browser** where you can view and edit data for each project.
```

Comment:
I thought that this sentence sounds a bit like Studio is *only* available as Data Browser, which is not the case. What do you think?

What would you think about saying that Client & Migrate are open source, instead of saying that Studio is not open source?

Reply:
Yes, you're right. I've now expanded the sentence to hopefully make it clearer.

My concern with highlighting Client & Migrate as open source and not saying anything about Studio (that's how I'm interpreting your suggestion at least) still leaves the question of the Prisma Studio status hanging, therefore, still unclear.

Just to make sure that I also cover this, here's how I would present that. Does it do justice to your suggestion?

Prisma is an [open source](https://github.com/prisma/prisma) next-generation ORM. It consists of the following parts:

* **Prisma Client**: An open-source, auto-generated, and type-safe query builder for Node.js & TypeScript
* **Prisma Migrate**: An open-source migration system
* **Prisma Studio**: GUI to view and edit data in your database

Reply:
That's something like I meant, but agree that it's not a great solution.

I like the updated version you have in the PR 👌